src/ModelTraits/SpatieMediaLibrary/HasMediaTrait.php

```diff
@@ -0,0 +1,188 @@
+<?php
+
+namespace Novius\Backpack\CRUD\ModelTraits\SpatieMediaLibrary;
+
+use Spatie\Image\Manipulations;
+use Spatie\MediaLibrary\HasMedia\HasMediaTrait as HasMediaTraitNative;
+use Spatie\MediaLibrary\Models\Media;
+
+/**
+ * Overrides the native HasMediaTrait trait to add some additional features :
+ * - adds some methods to work with medias stored as attributes (for example with the UploadableImage trait)
+ * - adds some properties and methods to easily handle conversions (crop and fit max)
+ *
+ * @package Novius\Backpack\CRUD\ModelTraits\SpatieMediaLibrary
+ */
+trait HasMediaTrait
+{
+    use HasMediaTraitNative;
+
+    /**
+     * The crop conversions
+     *
+     * @var array
+     */
+//    protected $mediaConversionsCrop = [
```

Comment:
We find already this information in the readme file. If it is intended as documentation I would put this within the trait phpdoc.